cli/cmd/sudo/project/get.go

```diff
@@ -44,6 +44,7 @@ func GetCmd(cfg *config.Config) *cobra.Command {
 			fmt.Printf("Prod OLAP DSN: %s\n", project.ProdOlapDsn)
 			fmt.Printf("Prod slots: %d\n", project.ProdSlots)
 			fmt.Printf("Prod deployment ID: %s\n", project.ProdDeploymentId)
+			fmt.Printf("Prod TTL: %d\n", project.ProdTtlSeconds)
```

Comment:
Update to `Prod hibernation TTL: %s` and then use the `time` package to format it as a human-readably number (instead of seconds)